.github/workflows/create-pr-compose-release.yml

```diff
@@ -0,0 +1,43 @@
+name: Create a automatic PR on Compose Release
+description: create a Pull Request with doc changes when a new version of Docker Compose is released
+
+on:
+  repository_dispatch:
+    types: [compose_release]
+
+jobs:
+  create_compose_doc_pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout docs source code
+        uses: actions/checkout@v3
+
+      - name: checkout Compose source code
+        with:
+          repository: docker/compose
+          ref: ${{ github.event.inputs.release }}
+          path: ./ComposeReleaseSource
+      - name: Prepare PR
+        run: |
+          rm -rf ./_data/compose-cli/*
+          cp -rf /.ComposeReleaseSource/docs/reference/*.yaml ./_data/compose-cli/
+          rm -rf /.ComposeReleaseSource
+          sed '/compose_version:/s/\".*\"/\"${{ github.event.inputs.release }}\"/g' _config.yml
+          git add -A .
```

Comment:
I think we could enhance this behavior and build from a remote reference instead so the logic in "Prepare PR" would be carry by compose repo.

Would be pretty much like this:

```yaml
name: Create a automatic PR on Compose Release
description: create a Pull Request with doc changes when a new version of Docker Compose is released

on:
  repository_dispatch:
    types: [compose_ref]

jobs:
  create_compose_doc_pr:
    runs-on: ubuntu-latest
    steps:
      -
        name: Checkout
        uses: actions/checkout@v3
      -
        name: Fetch content
        uses: docker/build-push-action@v3
        with:
          context: ${{ github.event.inputs.ref }}
          target: docs-dispatch
          output: ./_data/compose-cli
      - 
        name: Commit
        run: |
          git add -A .
      -
        name: Create PR
        uses: peter-evans/create-pull-request@v4
        with:
          commit-message: Update Compose reference API documentation to ${{ github.event.inputs.release }}
          signoff: true
          branch: update-compose-doc-${{ github.event.inputs.release }}
          delete-branch: true
          title: Update Compose reference API doc - ${{ github.event.inputs.release }}
          body: |
            Update the Compose reference API documentation to keep in sync with the latest release ${{ github.event.inputs.release }}
            
            - Auto-generated by [create-pull-request][1]
            [1]: https://github.com/peter-evans/create-pull-request
          labels: area/Compose
          draft: false
```